apps/src/templates/rubrics/rubrics.module.scss

```diff
@@ -1092,7 +1091,6 @@ p.evidenceLevelHeaderText {
   border: 1px solid $light_black;
   background: $neutral_white;
   gap: 10px;
-  padding: 3px 16px 0;
```

Comment:
Seemingly unrelated but a style fix where the suggested buttons had padding that the normal buttons did not. This padding is unnecessary and just makes the buttons wobble as you navigate learning goals.

Reply:
I thought I had caused it, but I think it was there before.